frontend/src/components/VModal/VModalContent.vue

```diff
@@ -180,8 +180,7 @@ export default defineComponent({
   @apply bg-tx;
 }
 .modal-backdrop {
-  /* linting plugin changes the order if we use non-custom value of h-screen for 100vh */
-  @apply h-[100dvh] h-[100vh];
```

Comment:
Is this comment no longer applicable?

Reply:
I think the comment is unclear, but it's still applicable.
Normally, I'd prefer to use `h-[100dvh] h-screen` so that if `dvh` is not supported, tailwind uses the `vh` value which has more support. However, then eslint reorders them to `h-screen h-[100dvh]`, which defeats the purpose. That's why I use the custom values for both 100dvh and 100vh.

Reply:
Should it be added back, but fixed, in that case? I'm not familiar enough with this to understand the implications either way, but it sounds like we're using this non-standard approach for a reason that could be documented (and prevent someone from thinking we should switch to the more standard approach)?